typescript/infra/config/environments/mainnet3/warp/configGetters/getRenzoPZETHWarpConfig.ts

```diff
@@ -91,6 +114,21 @@ export const getRenzoPZETHWarpConfig = async (): Promise<
                   },
                 ],
               },
+              hook: {
```

Comment:
could we maybe pull this into a function like `getRenzoHook(chain: ChainName)` that we share for pzETH and ezETH?

Reply:
added